src/include/CacheContent.hpp

```diff
@@ -0,0 +1,49 @@
+/**   
+ * MIT License
+ *
+ * Copyright (c) 2021 Tsuhao Chang
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+#ifndef CACHE_CONTENT_H_
+#define CACHE_CONTENT_H_
+
+#include <string>
+
+/**
+ * 包裝實際的檔案
+ * 讓其具有過期以及讀檔狀態的概念
+ */
+
+class CacheContent
+{
+public:
+    CacheContent();
+    ~CacheContent();
+    std::string *getContent();
```

Comment:
Why do you return a pointer to std::string? Do you expect the caller to modify its content?
If you want to avoid copying the string, you can return a const reference.

Reply:
Please add class comment.